Source/OxyPlot/Utilities/Helpers.cs

```diff
@@ -0,0 +1,27 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="Helpers.cs" company="OxyPlot">
+//   Copyright (c) 2020 OxyPlot contributors
+// </copyright>
+// --------------------------------------------------------------------------------------------------------------------
+
+namespace OxyPlot.Utilities
+{
+    /// <summary>
+    /// Provides general helper functions.
+    /// </summary>
+    public static class Helpers
+    {
+        /// <summary>
+        /// Switches the values of two specified variables.
+        /// </summary>
+        /// <typeparam name="T">The type of the variables.</typeparam>
+        /// <param name="value">The first value.</param>
+        /// <param name="other">The second value.</param>
+        public static void Switch<T>(this ref T value, ref T other) where T : struct
```

Comment:
I would oppose this being an extension mechanism on principle: there is no indication at the call-site that the `this` value is passed by reference, so the data-mutating semantics are unclear at a glance. In this case, I don't think there would be any loss of readability anyway, since neither parameter is 'more important'.

<hr />

I'd prefer it was called `Swap`, to distinguish it from the `switch` keyword and it's associated concepts.

<hr />

Is there a particular reason for the `T : struct` constraint?

Reply:
Regarding your first comment: Personally I've used this extension for a while and therefore it seems natural to me. But I can absolutely see how this could be confusing to others, so I'll change that to a regular method.

Second comment: Fine with me.

Third comment: The constraint is just because C# only allows ref extension methods on value types. Can be removed once this is a regular method.

Reply:
Oh yeah... I forgot about that detail completely. The new round of `ref` stuff makes me so sad.